src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/Example1.java

```diff
@@ -0,0 +1,19 @@
+/*
+AbstractClassName
+format = (default)^Abstract.+$
+ignoreModifier = (default)false
+ignoreName = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
```

Comment:
How do we know when to include package in examples and when not to?

Reply:
Parser will use Hardcoded list of checks to make a decision

Reply:
Seems like this should be some metadata added to test inputs, i.e. `includePackageDeclaration=true` or something.

Reply:
This can also be extended to things like imports, etc.

Reply:
I do not recommend this, it is required only for 1-3 Checks only.

Reply:
Imagine you are a contributor trying to create examples for your new check, but imports or package declarations aren't appearing in your examples. How can we guide someone to this hardcoded list? We should always keep contributor experience in mind.

Reply:
>How can we guide someone to this hardcoded list?

To be clear, this means by some documentation/test based on check type, etc

Reply:
Let's have simple hardcode solution now, issue for end of project, and at the end we will see all that we have and will be more clear how to handle it

Reply:
Moved to https://github.com/checkstyle/checkstyle/issues/13130